### DISCUSSION TITLE
Can't setup local dev environment due to role "user" does not exist ERROR

### DISCUSSION THREAD

**Discussion by tanvirraj** [outline/outline#3513] [General]
i'm following [this guide](https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/local-development-5hEhFRXow7) to setup outline locally but i failed with this error 

<img width="1038" alt="Screenshot 2022-05-10 at 7 28 44 PM" src="https://user-images.githubusercontent.com/3768120/167639801-080fdc68-3a5f-454b-a1d6-729d58d262dd.png">

can you give me any idea how can solve this issue

**Comment by almereyda** [comment]
This means your database has not been initialised with the same credentials, as those that you gave to Outline for connecting to it.

Please review your container environments for the Database and Outline container. If upstream documentation about the Container images does not spur insights, please feel free to respond here with any further questions.

**Reply by tanvirraj** [reply]
Thanks for your replay. 
the problem is i'm in mac, and if postgres  app running in 
<img width="984" alt="Screenshot 2022-05-11 at 5 29 19 PM" src="https://user-images.githubusercontent.com/3768120/167839365-a110a386-6f44-4d1a-b483-bffa225cb641.png">
my mac  i got this error `ERROR: role "user" does not exist`
if the postgres app not running in my mac 
i got this error 


i didn't change anything in file unless only add slack secret key for auth provider

**Reply by tanvirraj** [reply]
do i need to run a postgresql and redis   in mac separately? 
this guide assume i don't need to run those in mac 
https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/local-development-5hEhFRXow7
